tests/cli/test_cmds_mark_for_deployment.py

```diff
@@ -244,17 +255,19 @@ def on_enter_rolled_back_side_effect(self):
         deploy_group="test_deploy_group",
         commit="d670460b4b4aece5915caf5c68d12f560a9fe3e4",
         git_url="git://false.repo/services/test_services",
+        image_version="extrastuff",
     )
     mock_mark_for_deployment.assert_any_call(
         service="test_service",
         deploy_group="test_deploy_group",
         commit="old-sha",
         git_url="git://false.repo/services/test_services",
+        image_version="extrastuff",
```

Comment:
Since we're testing that we're going from `old-sha, None` to `FakeArgs.commit, FakeArgs.image_version` (`d670460b4b4aece5915caf5c68d12f560a9fe3e4, extrastuff`), this should be `image_version=None`.